src/bosh-director/lib/bosh/director/api/controllers/info_controller.rb

```diff
@@ -6,6 +6,7 @@ class InfoController < BaseController
       def initialize(config)
         super(config)
         @powerdns_manager = PowerDnsManagerProvider.create
+        @localdns_manager = LocalDnsManager.create(Config.root_domain, @logger)
```

Comment:
This call seems unnecessary as you've just accessed the root_domain below directly

Reply:
I've removed this line.